### PR TITLE
See you later tailwind

### LINKED ISSUE #446: Replace Tailwind with Pure CSS
Tailwind serves the goal of moving fast towards a first goal of having a minimum usable framework in place but it clearly shows its limitations in serving a framework like PyScript. This is mostly due to PyScript being a Framework and not an application. Here are a few of the problems:

* Tailwind is too big for a framework resulting either in a huge chunk of CSS to load 
* Not all users are interested in the visual components and they are currently paying a (high) cost for something they are not using
* PyScript should focus more on the flexibility of integrating well with applications and other frameworks. Tailwind is too invasive in that sense.

The goal is to replace Tailwind with pure CSS and maintain the overall look and feel of the current PyScript components.

### PR DESCRIPTION
**Pull request by fpliger**
Closes #446

### CODE REVIEW

**Comment by fpliger** [comment]
@marimeireles or anyone willing to pre-check this.... Mind a quick review and test of the examples? 🙂

**Comment by RSMuthu** [comment]
hi @fpliger, if this PR and issue is to clean `tailswindcss` dependency completely off, then i would suggest on removing the left outs as well. The following remnants will still be left out (as per the currently available commits):

- the `tailwindcss` dependency from the `package.json` file
- the complete `tailswind.config.js` config file 
- `tailwindcss` plugin setup on the rollup bundler configuration.

**Review by marimeireles — COMMENTED** [review]
It's breaking the CSS on `http://localhost:8000/pyscript/examples/repl2.html`.

Screenshot 2022-05-31 at 19 57 54

The output should be on the side but it's showing underneath every new cell.

Is there a reason why you didn't remove tailwind stuff? Like the ones addressed by @[RSMuthu](https://github.com/RSMuthu)?

Other than that. It works! :)
I tested all tests.

**Comment by fpliger** [comment]
@marimeireles no... mainly to check if it looks good to you and have more eyes checking the examples and other places I may have broken the code

**Comment by fpliger** [comment]
Thanks @RSMuthu  that's a great point :)

File: examples/repl.html [hunk 1]
```diff
@@ -7,8 +7,8 @@
     <title>REPL</title>
 
     <link rel="icon" type="image/png" href="favicon.png" />
-    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
-    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
+    <link rel="stylesheet" href="./build/pyscript_base.css" />
```
[the remark this answers is not in the document]
Comment (fpliger) [reply examples/repl.html]: Yeah... I messed up editing the wrong files while it was in progress.... Fixed

**Comment by fpliger** [comment]
Should have addressed comments from both @marimeireles and @RSMuthu . Ready for another round :)

**Review by marimeireles — COMMENTED** [review]
@fpliger this is so much CSS!
Thank you for the work here =)
It's a 👍 from me, other than these minor nitpicks.

**Comment by fpliger** [comment]
great catches, thank you @marimeireles . Should be good now

**Review by marimeireles — APPROVED** [review]

**Pull request merged**